dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/semantic/HeadwordRepositoryImpl.java

```diff
@@ -153,12 +154,25 @@ public BucketObjectsResponse<Headword> find(BucketObjectsRequest<Headword> bucke
     argumentMappings.put("startUuid", startUuid);
     argumentMappings.put("endUuid", endUuid);
 
+    String sortColumn = getSortColumn(bucketObjectsRequest);
+
+    // to avoid duplicate "hw.label, hw.label" if sortColumn = "label",
+    // just add sortColumn in other cases:
+    String columnNameLabel = getColumnName("label");
+    String selectForLabel = columnNameLabel;
```

Comment:
Da `columnNameLabel` nur hier verwendet wird, könnte man diese beiden Zeilen doch zusammenfassen.
```suggestion
    String selectForLabel = getColumnName("label");
```